packages/@lwc/compiler/package.json

```diff
@@ -45,10 +45,14 @@
         "@babel/core": "~7.21.8",
         "@babel/plugin-proposal-class-properties": "~7.18.6",
         "@babel/plugin-proposal-object-rest-spread": "~7.20.2",
+        "@babel/plugin-transform-async-to-generator": "~7.20.7",
+        "@babel/plugin-proposal-async-generator-functions": "~7.20.7",
+        "@locker/babel-plugin-transform-unforgeables": "0.19.6",
         "@lwc/babel-plugin-component": "2.45.5",
         "@lwc/errors": "2.45.5",
         "@lwc/shared": "2.45.5",
         "@lwc/style-compiler": "2.45.5",
         "@lwc/template-compiler": "2.45.5"
+
```

Comment:
You will need to rerun `node scripts/tasks/check-and-rewrite-package-json.js` because of this whitespace.

Reply:
No worries, I gotcha d3acb1afa